packages/language-support/package.json

```diff
@@ -47,7 +47,7 @@
     "vscode-languageserver-types": "^3.17.3"
   },
   "scripts": {
-    "gen-parser": "antlr4 -Dlanguage=TypeScript -visitor src/antlr-grammar/CypherParser.g4 src/antlr-grammar/CypherLexer.g4 -o src/generated-parser/ -Xexact-output-dir",
+    "gen-parser": "antlr4 -Dlanguage=TypeScript -visitor src/antlr-grammar/CommandLexer.g4 src/antlr-grammar/CommandParser.g4 -o src/generated-parser/ -Xexact-output-dir",
```

Comment:
This is a nitpick but could we find another name for the files? `CypherCmdParser`?